website/docs/migrations/migrate-to-v0.30.md

```diff
@@ -0,0 +1,74 @@
+# Migrate to Lumos v0.30
+
+## BREAKING: Buffer replaced by Uint8Array
+
+**Lumos can now run in browsers without any extra configs / polyfills with vite, webpack or create-react-app.**
+
+### `Buffer` was return type
+
+Methods / functions used to return a `Buffer` now returns a `Uint8Array`, e.g. In `@ckb-lumos/hd`
+
+```diff
+- export function privateToPublic(privateKey: Buffer | HexString): Buffer | HexString
++ export function privateToPublic(privateKey: Uint8Array | HexString): Uint8Array | HexString
+
+- export function mnemonicToSeedSync(mnemonic = "", password = ""): Buffer
++ export function mnemonicToSeedSync(mnemonic = "", password = ""): Uint8Array
+
+- export function mnemonicToSeed(mnemonic = "", password = ""): Promise<Buffer>
++ export function mnemonicToSeed(mnemonic = "", password = ""): Promise<Uint8Array>
+```
+
+You can use the `hexify` method from `@ckb-lumos/lumos/codec` to replace `buffer.toString('hex')`
+
+```diff
+- "0x" + privateToPublick(privKey).toString('hex')
++ hexify(privateToPublic(privKey))
+```
+
+**Notice:** `hexify` returns `HexString` which starts with `'0x'` while `buffer.toString('hex')` result has **no** `'0x'` prefix.
+
+Same for `@ckb-lumos/hd/KeyStore`
+
+```diff
+export default class Keystore {
+  ...
+- derivedKey(password: string): Buffer
++ derivedKey(password: string): Uint8Array
+  ...
+}
+```
+
+### `Buffer` was parameter type
+
+You can still pass in `Buffer` because `Buffer` is a subclass of `Uint8Array`, but using `Uint8Array` is recomended.
+
+`@ckb-lumos/hd/KeyChain`
+
+```diff
+export default class Keychain {
+  ...
+- constructor(privateKey: Buffer, chainCode: Buffer)
++ constructor(privateKey: Uint8Array, chainCode: Uint8Array)
```

Comment:
when `Buffer` is present in the parameter, there won't be any breaking change since it is [extended from `Uint8Array`](https://github.com/nodejs/node/blob/a0cb507ea67121392082795943a3dc00e5317b5d/doc/api/buffer.md#buffer). Thus, it's advisable to announce that these changes allow the APIs to use `Uint8Array`, and switching to `Uint8Array` is not mandatory

Reply:
<img width="926" alt="image" src="https://github.com/ckb-js/lumos/assets/7459812/1c992cfc-1d9c-4562-8e41-2f6c1926b015">
Yes. I did that.